src/other/utils.js

```diff
@@ -1,6 +1,4 @@
-const angular = require('angular-fix');
-
-export default {formlyEval, getFieldId, reverseDeepMerge, findByNodeName};
```

Comment:
I actually prefer to do all of my `export`s at the top of the file so I (and others) don't have to scroll to the bottom to find the important stuff. Could you update the PR to do this?